muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/index/body/continue-studies-panel.tsx

```diff
@@ -1,10 +1,19 @@
-import Link from "../../general/link";
 import * as React from "react";
-import { connect } from "react-redux";
+import {
+  displayNotification,
+  DisplayNotificationTriggerType,
+} from "~/actions/base/notifications";
```

Comment:
These are not used anywhere. Remove imports and props and mapDispatchTopProps definings